### PR TITLE
Edit and delete product features are finished.

### PR DESCRIPTION
**Pull request by starkshaw**

### CODE REVIEW

File: app/views/pages/dashboard.html.haml [hunk 1]
```diff
@@ -1,9 +1,26 @@
...
+
+      %h5
+        = link_to shop.name, shop_path(shop)
+      - if shop.products.empty?
+        %h6 No product in this shop.
+        = link_to 'Create product in this shop', new_shop_product_path(shop)
+      - else
+        %div
+          %h6 PRODUCTS
+          - shop.products.each do |pro|
```
Comment (danielpaul) [inline app/views/pages/dashboard.html.haml]: use more readable variable names? `pro` → `product`

Comment (starkshaw) [inline app/views/pages/dashboard.html.haml]: You picky bastard... :angry:

Comment (danielpaul) [inline app/views/pages/dashboard.html.haml]: It's for future readability :P Good coding practices :P

Comment (danielpaul) [inline app/views/pages/dashboard.html.haml]: Haha it's also my job to be picky and a perfectionist :P

**Comment by starkshaw** [comment]
I hereby request my wage... :anger: :angry:

**Comment by danielpaul** [comment]
hahaha. @starkshaw need to be descriptive on commit message :P :P

**Pull request merged**